atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -81,7 +81,8 @@ def __init__(self, tasker, workflow, labels,
             raise RuntimeError("labels have to be dict")
         self.labels = labels
         self.dont_overwrite = dont_overwrite
-        self.aliases = aliases or {}
+        default_aliases = {k: v for k, v in _PREFERRED_LABELS}
```

Comment:
This looks the wrong way round.

Also we need to make sure all these are covered:
https://github.com/projectatomic/ContainerApplicationGenericLabels/blob/master/vendor/redhat/labels.md

as well as any we use from here:
https://github.com/projectatomic/ContainerApplicationGenericLabels/blob/master/README.md